ocaml/xapi/xapi_gpumon.ml

```diff
@@ -15,7 +15,7 @@
 module D = Debug.Make(struct let name="xapi" end)
 open D
 
-let service = "/sbin/service"
+let systemctl = "/usr/bin/systemctl"
 let gpumon = "xcp-rrdd-gpumon"
 let pidfile = "/var/run/xcp-rrdd-gpumon.pid"
```

Comment:
I'm starting to wonder whether this is a good idea. We're duplicating the pidfile definition from the one in the initscript (soon to be systemd service). Why don't we make `check` use `systemctl is-active [-q] xcp-rrdd-gpumon` which exits 0 if it is active and non-zero if not.